CONTRIBUTING.md

```diff
@@ -1,303 +1,14 @@
-# Contributing
+# Optimism monorepo contributing guide
 
-🚨 Before making any non-trivial change via Pull Request, please first open an issue 
-describing the change to solicit feedback and guidance. This will increase the 
-likelihood of your Pull Request getting merged.
+**Please read our [general contributing guide](https://github.com/ethereum-optimism/.github/blob/master/CONTRIBUTING.md) before continuing**
 
-## Contributing in Issues
 
-For any issue, there are fundamentally three ways an individual can contribute:
+These packages only require 1 reviewer (all other packages require 2)
```

Comment:
```suggestion
## PR Review Requirements

These packages only require 1 reviewer (all other packages require 2 reviewers, unless the changes do not affect production or test code). 
```

That's the intent of the guidelines, does that phrasing make sense?

Reply:
added!